### PR TITLE
feat(*): add world-private icon

### PR DESCRIPTION
**Pull request by portikM**
# Summary

1. Adds `world-private` icon
2. In sandbox adds search query to route query params to make search results sharable (e.g. `https://kong.github.io/icons/?search=World`)

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Review by DariaYeremina — APPROVED** [review]

File: sandbox/components/PageHeader.vue
Comment (adamdehaven) [inline sandbox/components/PageHeader.vue:44]: You need to add a check here to make sure the query doesn't already exist so that it doesn't continuously route

Comment (portikM) [reply sandbox/components/PageHeader.vue:44]: Ah fair enough

Comment (portikM) [reply sandbox/components/PageHeader.vue:44]: Warning: making mistakes in code can summon Adam any time of day

Comment (adamdehaven) [reply sandbox/components/PageHeader.vue:44]: Just like to help 😅 (in an Uber trying to ignore the music lol)

Comment (portikM) [reply sandbox/components/PageHeader.vue:44]: Fixed

File: sandbox/components/PageHeader.vue
Comment (jillztom) [inline sandbox/components/PageHeader.vue:49]: Is‘t it going to send me to home page if both query string and search query is same?

Comment (portikM) [reply sandbox/components/PageHeader.vue:49]: Yes and now. It does save the search query in input field but removes the query from route (screenshot)

Screen Shot 2023-09-20 at 3 56 43 PM

File: sandbox/components/PageHeader.vue
Comment (jillztom) [reply sandbox/components/PageHeader.vue:44]: This is all it requires:

```
watch(query, (searchQuery: string) => {
  emit('search', searchQuery)

  if (searchQuery) {
    router.push({ name: 'home', query: { search: searchQuery } })
  }
}, { immediate: true })
```
It's already checking if the query exists and then redirects them to the correct query param. I don't see a potential infinite loop 🤔

Comment (adamdehaven) [reply sandbox/components/PageHeader.vue:44]: It's going to push every time the query string changes, which will also make the component remount every time. At a minimum it needs a debounce on the search string

Comment (jillztom) [reply sandbox/components/PageHeader.vue:44]: yes, debounce is a valid point. 

If we support search through query param, it needs to be reflected on the url too

Comment (portikM) [reply sandbox/components/PageHeader.vue:44]: If I didn't have `else` block it doesn't remove the query when search is empty (recording)

https://github.com/Kong/icons/assets/36751160/5ae7617d-0d57-486d-bbf1-5a321991eb87

File: sandbox/components/PageHeader.vue
Comment (portikM) [reply sandbox/components/PageHeader.vue:49]: Let's merge this already 😅

File: sandbox/components/PageHeader.vue
Comment (jillztom) [reply sandbox/components/PageHeader.vue:44]: Screenshot 2023-09-20 at 4 05 27 PM

Understood, what I was saying is the query param is lost when we share the link and opens in a window. 

Anyways, this is not a deal breaker in any ways. We can fix it later if it's an issue.

**Review by jillztom — APPROVED** [review]

File: sandbox/components/PageHeader.vue
Comment (adamdehaven) [reply sandbox/components/PageHeader.vue:44]: Yes, but I would say don't merge this sandbox change if it's not completely functional 🙏

**Pull request merged**
